### ISSUE TITLE
[Feature Request] Sort wishlist by want (favorite picker)

### ISSUE THREAD

**Issue by Rukishou** [tfedor/AugmentedSteam#928]
Would be useful, especially for people with long wishlists that are tedious or implausible to sort manually, to have something like a [favorite picker](https://www.dragonflycave.com/favorite.html) giving 2+ options and after selecting those you want most it gets sorted for you (w/ option to manually fine tune after of course).

**Comment by MxtOUT** [comment]
Uh, neat one!

**Comment by Revadike** [comment]
Indeed, I like this idea. It would make sorting big lists more bearable. Though, I can imagine some comparisons would be hard to decide upon, so a skip option should be there.
